### PR TITLE
EDL: fixed IPv6 improper exports

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35549

## Description
apply URL filtering on non-IP indicators only

### CODE REVIEW

File: Packs/EDL/Integrations/EDL/EDL_test.py [hunk 1]
```diff
@@ -213,6 +213,28 @@ def test_create_values_for_returned_dict(self):
...
+    @pytest.mark.validate_basic_authentication
+    def test_create_values_for_returned_dict__filters(self):
+        from EDL import create_values_for_returned_dict, EDL_VALUES_KEY, RequestArguments
+        iocs = [
+            {'value': '2603:1006:1400::/40', 'indicator_type': 'IPv6'},
+            {'value': '2002:ac8:b8d:0:0:0:0:0', 'indicator_type': 'IPv6'},
+            {'value': 'demisto.com:369/rest/of/path', 'indicator_type': 'URL'},
+            {'value': 'panw.com/path', 'indicator_type': 'URL'},
+            {'value': '*.domain.com', 'indicator_type': 'URL'},
+            # {'value': '', 'indicator_type': 'IPv6'},
```
Comment (teizenman) [inline Packs/EDL/Integrations/EDL/EDL_test.py:225]: Remove?

File: Packs/EDL/Integrations/EDL/EDL.py [hunk 2]
```diff
@@ -308,28 +308,29 @@ def create_values_for_returned_dict(iocs: list, request_args: RequestArguments)
...
-            continue
-        # Reformatting to to PAN-OS URL format
-        with_invalid_tokens_indicator = indicator
-        # mix of text and wildcard in domain field handling
-        indicator = _INVALID_TOKEN_REMOVAL.sub('*', indicator)
-        # check if the indicator held invalid tokens
-        if with_invalid_tokens_indicator != indicator:
-            # invalid tokens in indicator- if drop_invalids is set - ignore the indicator
-            if request_args.drop_invalids:
+        if ioc_type not in ['IP', 'IPv6']:
```
Comment (DeanArbel) [inline Packs/EDL/Integrations/EDL/EDL.py:311]: @roysagi @hod-alpert shouldn't port stripping be applicable to IPs as well?
https://stackoverflow.com/a/186848/9827323

Comment (teizenman) [inline Packs/EDL/Integrations/EDL/EDL.py:311]: 1. You can use the constants from CSP for it.
2. Do you need to add CIDRs as well?

**Review by teizenman — CHANGES_REQUESTED** [review]

File: Packs/EDL/Integrations/EDL/EDL.py (hunk 2 above)
Comment (yaakovi) [reply Packs/EDL/Integrations/EDL/EDL.py:311]: in the case you are referring the indicator will be marked as URL, not IP/IPv6

**Review by teizenman — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi @teizenman @hod-alpert @DeanArbel @roysagi  Release Notes were reviewed.

**Pull request merged**